portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -10,43 +10,56 @@ import cromwell.pipeline.datastorage.dto.auth.AccessTokenContent
+
   implicit val system: ActorSystem = ActorSystem("cromwell-pipeline")
   implicit val executionContext: ExecutionContextExecutor = system.dispatcher
 
-  val log = LoggerFactory.getLogger(CromwellPipelineApp.getClass)
-  val components = new ApplicationComponents()
-
-  import components.applicationConfig.webServiceConfig
-  import components.controllerModule._
-  import components.datastorageModule._
-
-  pipelineDatabaseEngine.updateSchema()
-
-  type SecuredRoute = AccessTokenContent => Route
-  def routeCombiner(routes: SecuredRoute*): SecuredRoute = token => concat(routes.map(_(token)): _*)
-
-  implicit val rejectionHandler: RejectionHandler = RejectionHandler
-    .newBuilder()
-    .handle {
-      case ValidationRejection(msg, _)      => complete(StatusCodes.BadRequest, msg)
-      case MissingAccessTokenRejection(msg) => complete(StatusCodes.Unauthorized, msg)
-    }
-    .result()
-    .withFallback(RejectionHandler.default)
-
-  val route = authController.route ~ securityDirective.authenticated {
-    routeCombiner(
-      userController.route,
-      projectController.route,
-      projectFileController.route,
-      runController.route,
-      configurationController.route
-    )
-  }
+  Try {
+
+    val components: ApplicationComponents = new ApplicationComponents()
+
+    import components.applicationConfig.webServiceConfig
+    import components.controllerModule._
+    import components.datastorageModule._
+
+    pipelineDatabaseEngine.updateSchema()
 
-  log.info(s"Server online at http://${webServiceConfig.interface}:${webServiceConfig.port}/")
-  Http().newServerAt(webServiceConfig.interface, webServiceConfig.port).bindFlow(route)
+    type SecuredRoute = AccessTokenContent => Route
+
+    def routeCombiner(routes: SecuredRoute*): SecuredRoute = token => concat(routes.map(_(token)): _*)
+
+    implicit val rejectionHandler: RejectionHandler = RejectionHandler
+      .newBuilder()
+      .handle {
+        case ValidationRejection(msg, _)      => complete(StatusCodes.BadRequest, msg)
+        case MissingAccessTokenRejection(msg) => complete(StatusCodes.Unauthorized, msg)
+      }
+      .result()
+      .withFallback(RejectionHandler.default)
+
+    val route = Route.seal(authController.route ~ securityDirective.authenticated {
+      routeCombiner(
+        userController.route,
+        projectController.route,
+        projectFileController.route,
+        runController.route,
+        configurationController.route
+      )
+    })(rejectionHandler = rejectionHandler)
+
+    log.info(s"Server online at http://${webServiceConfig.interface}:${webServiceConfig.port}/")
+    Http().newServerAt(webServiceConfig.interface, webServiceConfig.port).bindFlow(route)
+
+  } match {
+    case Success(_) =>
+      log.info(s"Application is running")
+    case Failure(e) =>
+      system.terminate()
+      log.error(s"Application cannot be started.\n${e.fillInStackTrace()}")
```

Comment:
When you log at `error` level, you provide text description and an error as a second argument. No need to interpolate Exception to string.